src/components/Explore/Explore.tsx

```diff
@@ -370,6 +370,7 @@ const Explore: React.FunctionComponent<{
                 )
               }
               quote={getSocialQuote(selectedLocations, currentMetric)}
+              region={region ? region : undefined}
```

Comment:
you can just pass `region={region}`

the region variable in Explore.tsx [is grabbed here](https://github.com/covid-projections/covid-projections/blob/develop/src/components/Explore/Explore.tsx#L115) -
`const region = useRegionFromParams();`

`useRegionFromParams` will return `null` if the region comes up undefined ([see the source code here](https://github.com/covid-projections/covid-projections/blob/develop/src/common/regions/region_hooks.ts#L17))

Reply:
Yea, strangely enough when I tried that I get the following error:
`Type 'Region | null' is not assignable to type 'Region | undefined'.
  Type 'null' is not assignable to type 'Region | undefined'.`
So I implemented as such. I was hoping there would be a better solution though 😕

Reply:
just chatted about this on a call but lmk if there are more typescripty questions!

Reply:
After additional discussion with @pnavarrc, we figured it's probably better to just leave the type check as is. Changing it may trigger quite a significant amount of changes which we think may not be worthwhile to accommodate. Let us know if you have any additional thoughts about this though! 🙂